prow/plugins/config.go

```diff
@@ -896,8 +887,8 @@ func (c *Configuration) TriggerFor(org, repo string) Trigger {
 }
 
 func (t *Trigger) SetDefaults() {
-	if t.TrustedOrg != "" && t.JoinOrgURL == "" {
-		t.JoinOrgURL = fmt.Sprintf("https://github.com/orgs/%s/people", t.TrustedOrg)
+	if t.JoinOrgURL == "" {
+		t.JoinOrgURL = fmt.Sprintf("https://github.com/orgs/%s/people", t.JoinOrgURL)
```

Comment:
Hmmm, it does not seem to be right. Resulting string will be `https://github.com/orgs//people`

Reply:
@deepsm007 you have not corrected this one